news/_posts/translation/2020-07-10-do-you-speak-prestashop-june-2020.md

```diff
@@ -0,0 +1,126 @@
+---
+layout: post
+title:  "Do you speak PrestaShop? – June 2020 edition"
+subtitle: "Where you'll learn that Esperanto is on the rise"
+date:  2020-07-10 14:30:11
+authors: [ JuneByun ]
+icon: icon-world
+image: /assets/images/2019/01/Build-Crowdin-banner.jpg
+tags: [translation, i18n, l10n]
+---
+
+Contributing to PrestaShop is not only about the code, it's also about taking part in the [PrestaShop translation project](https://crowdin.com/project/prestashop-official)! This report will tell you how the translations of the software evolved in May.
+
+![Crowdin Monthly banner](/assets/images/2019/01/Build-Crowdin-banner.jpg)
+
+## Project news
+
+
+### First news, Esperanto is on the rise :fire:
+
+I am thrilled to let you know that Esperanto has been on the rise since the past month. From the beginning of May to the end of June, the Esperanto translation project went from 25% to 34% words translated, that is to say a 9% evolution. I wish to take this opportunity to thank mirmal and Milan Kolka (milankolka) for your work! :heart_eyes: 
+
+Many of you dedicate their time to contribute to the PrestaShop projects on Crowdin but cannot validate translations because you don’t have the rights. I thus wanted to tell you to not hesitate in [asking me](https://crowdin.com/profile/june.byun) if you wish to become a proofreader. It would allow you to suggest and approve the strings you and others translate.
+
+### Daily language packs updates
+
+In the [last monthly report](https://build.prestashop.com/news/do-you-speak-prestashop-may-2020/), I explained how the translation packs are updated in PrestaShop now. What has to be kept in mind is that only translations of the 1.7.7 PrestaShop version translation projects can be updated and that it has to be manually exported and uploaded to one’s PrestaShop. 
+
+But the good news is that an issue has been created to solve this inconvenience and to make all translations available for PrestaShops, regardless of their version. :sparkles: You can follow the progress of this issue [here](https://github.com/PrestaShop/PrestaShop/issues/20011).
```

Comment:
```suggestion
But the good news is that an issue has been created to start finding a solution to this inconvenience. You can follow the progress, or even involve if you feel like it - it would be a great contribution -, just [here](https://github.com/PrestaShop/PrestaShop/issues/20011).
```